xds/internal/resolver/xds_resolver.go

```diff
@@ -97,18 +103,18 @@ func (b *xdsResolverBuilder) Build(t resolver.Target, cc resolver.ClientConn, rb
 // contains DialCreds, we use it as is. If it contains a CredsBundle, we use
 // just the transport credentials from the bundle. If we don't find any
 // credentials on the parent channel, we resort to using an insecure channel.
-func defaultDialCreds(balancerName string, rbo resolver.BuildOptions) grpc.DialOption {
+func defaultDialCreds(balancerName string, rbo resolver.BuildOptions, r *xdsResolver) grpc.DialOption {
```

Comment:
This is also unfortunate that we have to change the signature of the function just so that we can pass in the logging prefix. Would it be better to just make this function a method in the `xdsResolver` type instead?